captum/optim/_core/optimization.py

```diff
@@ -56,15 +56,17 @@ def __init__(
                         optimization.
             lr (float): The learning rate to use with the Adam optimizer.
         """
-        self.model = model
+        self.model = model or nn.Identity()

```

Comment:
add `optional` to the type in the corresponding doc (comments above) as well? btw, what is the use case of not giving a `model`

Reply:
@aobo-y I thought that listing optional in the docs meant that there was a preset value for the variable? I've added it now though.

The reasoning behind making the model optional is that for example in some cases the image parameterization may be a generative model. Thus it doesn't make a lot of sense to require a second model be used in these cases unless the user wants to link them.